src/module-elasticsuite-core/Search/Context.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * DISCLAIMER
+ * Do not edit or add to this file if you wish to upgrade Smile Elastic Suite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCore
+ * @author    Romain Ruaud <romain.ruaud@smile.fr>
+ * @copyright 2018 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+namespace Smile\ElasticsuiteCore\Search;
+
+use Magento\Store\Model\StoreManagerInterface;
+
+/**
+ * Elasticsuite Search Context
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteCore
+ * @author   Romain Ruaud <romain.ruaud@smile.fr>
+ */
+class Context
```

Comment:
The implemented method are OK but I want to be able to write arbitrary data in the context.
Maybe add customer_group by default in the context.